views/NewChart.react.js

```diff
@@ -26,17 +26,14 @@ const initialValues = (user) => ({
       targetValues: targetValuesFields(user.userAccess),
     },
   ],
+  public: false,
 })
 
 const NewChart = ({ classes, user }) => {
   const handleSubmit = async (e, formValues) => {
-    try {
-      e.preventDefault()
-      const { data } = await createChart(formValues)
-      window.location.assign(routes.chart(data.chart_id))
-    } catch (error) {
-      console.error(error)
-    }
+    e.preventDefault()
+    const { data } = await createChart(formValues)
+    window.location.assign(routes.chart(data.chart_id))
```

Comment:
why was the try catch removed?

Reply:
We wanted to have errors be loud and with a catch we might only log them or swallow them.

Reply:
hmm, that makes sense to me for things like creating something in the db, but I _think_ this is an api call? should the view fail gracefully if the network call fails, and the relevant server code could fail loudly?

Reply:
For now we want to know of any errors that happen until we implement something flashier to display an error on screen.

Reply:
Gotcha, one option would be catch the error and use a browser alert to inform the user. The network tab would have the error. You have more context here 😄  so if that doesn't make sense to do, def keep it as is 👍